apollo-client/src/main/java/com/ctrip/framework/apollo/monitor/internal/listener/impl/DefaultApolloClientNamespaceApi.java

```diff
@@ -182,7 +184,8 @@ public Map<String, NamespaceMetricsString> getNamespaceMetricsString() {
     namespaces.forEach((namespace, metrics) -> {
       NamespaceMetricsString namespaceMetricsString = new NamespaceMetricsString();
       namespaceMetricsString.setFirstLoadTimeSpendInMs(metrics.getFirstLoadTimeSpendInMs());
-      namespaceMetricsString.setLatestUpdateTime(metrics.getLatestUpdateTime().toString());
+      Optional<String> date = DateUtil.formatLocalDateTime(metrics.getLatestUpdateTime());
+      date.ifPresent(namespaceMetricsString::setLatestUpdateTime);
```

Comment:
No local variable will be better?

DateUtil.formatLocalDateTime(metrics.getLatestUpdateTime())
.ifPresent(namespaceMetricsString::setLatestUpdateTime);